pyaedt/modules/Boundary.py

```diff
@@ -2962,7 +2960,6 @@ def angle(self):
 
     @angle.setter
     def angle(self, angle=None):
-        """Set the port angle."""
         self._logger.warning("Angle can not be modified. Bug in AEDT API.")
```

Comment:
```suggestion
        self._logger.warning("Angle cannot be modified. Bug in AEDT API.")
```
Do we want to use "bug" or rewrite, perhaps "A problem exists in the AEDT API." or "This capability has not yet been implemented in the AEDT API."